packages/react/src/components/tabs/Tabs.stories.tsx

```diff
@@ -74,3 +74,38 @@ export const Small = () => (
     </TabPanel>
   </Tabs>
 );
+
+export const WithCustomTheme = () => {
+  const theme = {
+    '--tab-color': 'var(--color-brick)',
```

Comment:
We could use some less bright color in the example 😎

Reply:
What do you mean? There are not many brand colours you can actually use in that example. There are not many brand colours that meet the `3:1` contrast requirement on UI elements.

Reply:
I meant the red. It looks like there is an error in tabs

Reply:
- [ ] Improve custom theme colors; align colors with doc site